engine/src/main/java/org/hibernate/search/spatial/SpatialFieldBridgeByHash.java

```diff
@@ -54,7 +56,7 @@ public void configureFieldMetadata(String name, FieldMetadataBuilder builder) {
 		super.configureFieldMetadata( name, builder );
 		if ( spatialHashIndex ) {
 			for ( int i = topSpatialHashLevel; i <= bottomSpatialHashLevel; i++ ) {
-				builder.field( SpatialHelper.formatFieldName( i, name ), FieldType.STRING );
+				builder.field( hashIndexedFieldNames[i], FieldType.STRING );
 			}
 		}
 		if ( numericFieldsIndex ) {
```

Comment:
`latitudeIndexedFieldName` and `longitudeIndexedFieldName` could be re-used here.